Marlin/src/gcode/calibrate/G34_M422.cpp

```diff
@@ -152,17 +163,19 @@ void GcodeSuite::G34() {
       SERIAL_ECHOLNPAIR("\nITERATION: ", int(iteration + 1));
 
       // Initialize minimum value
-      float z_measured_min = 100000.0f;
+      float z_measured_min = 100000.0f,
+            z_measured_max = 0.0;
```

Comment:
I should have initialized this to a large negative number.
```suggestion
            z_measured_max = -100000.0f;
```